app/views/katello/api/v2/subscriptions/base.json.rabl

```diff
@@ -13,3 +13,12 @@ attributes :sockets, :cores, :ram
 attributes :instance_multiplier, :stacking_id, :multi_entitlement
 attributes :type
 attributes :name => :product_name
+attributes :unmapped_guest
```

Comment:
I was just following the previous lazy accessor attribute here http://git.io/vW59b (from pre subscriptions scoped search) I can update the column to unmapped_guests_only and change where it is used. What about the API though?